File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task128_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Random;

public class Main {
    public static void main(String[] args) {
        String lettersAndDigits = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
        StringBuilder token = new StringBuilder();
        Random random = new Random();
        for (int i = 0; i < 10; i++) {
            token.append(lettersAndDigits.charAt(random.nextInt(lettersAndDigits.length())));
        }
        System.out.println("Random token is " + token.toString());
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task128_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <string>
#include <cstdlib>
#include <ctime>

std::string generateToken(int length) {
    std::string lettersAndDigits = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
    std::string token;
    for (int i = 0; i < length; i++) {
        token += lettersAndDigits[rand() % lettersAndDigits.length()];
    }
    return token;
}

int main() {
    srand(time(0));
    std::cout << "Random token is " << generateToken(10) << std::endl;
    return 0;
}